### ISSUE TITLE
 Problem loading modules with systemJS (SPO)

### ISSUE THREAD

**Issue by MarianoPrz** [SharePoint/sp-dev-docs#1752]
#### Category
- [x] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

Hello, I have a problem using systemJS to load modules in Sharepoint online, yesterday it worked fine, but now there seems to be a problem with the authentication flow. Only in Internet Explorer works well. This problem I have had in two different tenants. Microsoft changed something at the level of sharepoint?

Thanks in advance.

![image](https://user-images.githubusercontent.com/10661704/39245892-466aebdc-4895-11e8-93f2-9decd71ff125.png)

**Comment by VesaJuvonen** [comment]
Hi Mariano, we would need to have slightly more information to start resolving this. Any chance you could provide correlation ID from the time when the issue is happening so that we can check the situation from the logs. Thx.

**Comment by jhholm** [comment]
I do not think this is a CORS issue. As I can see you are using systemjs 0.20 which had some changes. If you're loading from SharePoint Online, you are going to need to be authenticated, thus authentication cookie must exist.

Please check this issue:
https://github.com/systemjs/systemjs/issues/1731

**Comment by marcmoraRAONA** [comment]
But then, what was changed on Sharepoint online ? because It worked fine in this way since Nov. 2017 and one week ago this problem appeared

**Comment by VesaJuvonen** [comment]
The problem here is that unless we can reproduce this, there's really no way for us to investigate as this does not seem to be a generic problem. Is there any chance you could share your example code for us to find a repro step?

**Comment by marcmoraRAONA** [comment]
We solved the problem changing version of SystemJs. If we used the last version , the problem appears, if we used the same version that uses SPFX  (0.19), then works fine.

To reproduce the problem:
If you load some javascript in your custom masterpage using last version of SystemJS and this javascript is on a Sharepoint Library, the problem appears. If you do the same but you put javascript outside sharepoint (like CDN or Azure Storage ), then works fine.
